src/hooks/transactions/batch/tracker/useCheckHangingBatchesFallback.ts

```diff
@@ -1,13 +1,11 @@
 import { useCallback, useEffect } from 'react';
-import {
-  AVERAGE_TX_DURATION_MS,
-  TRANSACTIONS_STATUS_DROP_INTERVAL_MS
-} from 'constants/transactionStatus';
-import { extractSessionId } from 'hooks/transactions/helpers/extractSessionId';
-import { timestampIsOlderThan } from 'hooks/transactions/helpers/timestampIsOlderThan';
+import { TRANSACTIONS_STATUS_DROP_INTERVAL_MS } from 'constants/transactionStatus';
 import { removeBatchTransactions } from 'services/transactions';
 import { getTransactionsStatus } from 'utils/transactions/batch/getTransactionsStatus';
 import { sequentialToFlatArray } from 'utils/transactions/batch/sequentialToFlatArray';
+import { extractSessionId } from '../../helpers/extractSessionId';
```

Comment:
absolute?

Reply:
It is in the same folder/contex. Relative is better